src/index.js

```diff
@@ -0,0 +1,79 @@
+import './style.css';
+
+class TaskList {
+  constructor() {
+    this.size = 0;
+    this.ul = document.getElementById('task-list');
+    this.savedList = JSON.parse(localStorage.getItem('savedList')) || [];
+    // this.displayAllTask();
```

Comment:
- [ ] Remember to always remove commented code

Reply:
Done